TABTestKit/Classes/Biometrics/Biometrics.h

```diff
@@ -0,0 +1,22 @@
+//
+//  Biometrics.h
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 31/10/2018.
+//  Copyright © 2018 kane.codes. All rights reserved.
```

Comment:
You can change `kane.code` to The App Business in these :)

Reply:
Done!